cdap-docs/faq/source/cdap-startup.rst

```diff
@@ -0,0 +1,226 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :copyright: Copyright © 2015 Cask Data, Inc.
+
+:titles-only-toc: true
+
+.. _faq-installation-startup:
+
+======================
+CDAP FAQ: CDAP Startup
+======================
+
+.. contents::
+   :depth: 2
+   :local:
+   :backlinks: entry
+   :class: faq
+
+
+Startup
+=======
+
+My CDAP install on CDH using Cloudera Manager doesn't startup |---| what do I do?
+---------------------------------------------------------------------------------
+We have a :ref:`tutorial <step-by-step-cloudera-add-service>` with instructions on how to install CDAP on CDH 
+(`Cloudera Data Hub <http://www.cloudera.com/content/www/en-us/resources/datasheet/cdh-datasheet.html>`__) 
+using `Cloudera Manager <http://www.cloudera.com/content/www/en-us/products/cloudera-manager.html>`__. 
```

Comment:
Who is "we"? Other docs are more third person...

`Instructions for installing CDAP on CDH using Cloudera Manager are available at...` or something... I don't mind using "you" to refer to the reader, but feel that "we" should be replaced with simply "CDAP" or "The" where appropriate. (I said feel, so feel free to ignore, too... ;-P)